### ISSUE TITLE
Ties in standings glitch?

### ISSUE THREAD

**Issue by psychoned** [dave0/leaguerunner#78]
Just wondering if there is a glitch in the way ratings ties are listed in the rankings... twice this year our team has been tied in rankings points with a team and have been ranked below them despite having more wins, more points for, fewer points against and a better +/-, not to mention having beaten them in a head to head game. It matters not for general week to week standings, but when it comes to splitting the ladder into playoff pools it could have an effect.

Cheers,
Ken

**Comment by dave0** [comment]
Nope, it's not a glitch.  For ladder leagues, the ranking criteria are, as chosen by OCUA:
- Rating
- SOTG
- wins
- +/-
- points for

For non-ladder leagues, the criteria are:
- Wins
- Head-to-head wins
- SOTG
- +/-
- fewest number of losses

I've forgotten the historical reason for those tiebreaker values, but checking the git history it looks like it's been that way since summer 2007. 

Unfortunately, the "Summer League Handbook" page that contained the tiebreaker info on ocua.ca seems to have been nuked in the last site redesign.  I'll leave this ticket open and document the tiebreaker criteria within Leaguerunner as a popup help item when I get time, so it can't accidentally be deleted by wayward web designers.

**Comment by psychoned** [comment]
Well, thanks for the clarification, Dave.  I think it would be good to add
the tiebreaker documentation, but certainly no rush on that.

Also, I was thinking the other day that it might be nice to add a
"Nickname" field to the account, that people could fill in if they desired,
and would show up on roster lists and such in quotations if it was
populated, such as Paul "PMo" Morrison, or Steve "Wheels" Penny.... There
are a number of people in the league that eventually develop a nickname
that sticks, and invariably newer people just hear their nickname and never
hear their real name. So this would add some clarity.

Again, just an idea, and no hurry... but thought I'd mention it.

Hope all is well,
Ken
